inch.go

```diff
@@ -313,21 +321,31 @@ func (s *Simulator) generateBatches() <-chan []byte {
 			fields = append(fields, []byte(fmt.Sprintf("v%d=1%s", i, delim))...)
 		}
 
+		// Size internal buffer to consider mx+tags+ +fields.
+		buf := bytes.NewBuffer(make([]byte, 0, 2+len(tags)+1+len(fields)))
```

Comment:
mx=2 is a poor value for >9 measurements. `len(tags)` is always 0 at this point.